easynlp/core/trainer.py

```diff
@@ -115,7 +115,7 @@ def resume_from_ckpt(self, model_module, args):
         model_file = args.resume_from_checkpoint + '.bin'
         if 'oss://' in args.resume_from_checkpoint:
             local_file = 'easynlp_resume_pytorch_model.meta.bin'
-            io.download(model_file, local_file)
+            io.download(meta_file, local_file)
```

Comment:
meta_file not defined?

Reply:
should be "meta_file" here, but "model_file" was written twice : )

Reply:
nice catch~